checker/tests/nullness-extra/issue309/Makefile

```diff
@@ -12,3 +11,7 @@ all:
 	diff -u Expected.txt Out.txt
 	rm Out.txt lib.jar bin/*.class bin/lib/*.class
 	rmdir bin/lib bin/
+
+clean:
+	rm -f Out.txt lib.jar bin/*.class bin/lib/*.class
+	rmdir bin/lib bin/
```

Comment:
In CI, I think one of these targets causes a failure, because all calls clean which rmdirs a directory that doesn't exist, leading to an error.

Reply:
Remove clean call for `all` target.